src/Common/tests/Data/TinyAssembly.il

```diff
@@ -0,0 +1,2 @@
+.assembly TinyAssembly {}
```

Comment:
Can we somehow build TinyAssembly.dll as part of the build process rather than checking in the binary?